ui/app/src/state/epics/preview.ts

```diff
@@ -35,7 +28,7 @@ export default [
       withLatestFrom(state$),
       tap(([{ type, payload }, state]) => {
         if (payload) {
-          setStoredPreviewToolsPanelPage(state.sites.active, payload);
+          setPreferences({ toolsPanelPage: JSON.stringify(payload) }, state.sites.active).subscribe(() => {});
         }
       }),
       ignoreElements()
```

Comment:
Best if we switchMap these so we can map to other actions or error. Same for all of the kind.

Reply:
close